src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: function (num) {
+      result += num;
+
+      return this;
+    },
```

Comment:
Instead of using the `function` keyword, you can use the method shorthand syntax for defining `add`.